app/components/button.jsx

```diff
@@ -96,6 +97,11 @@ function Button(props) {
         onKeyDown={handleKeyDown}
         {...otherProps}
       >
+        {value === 'create' && (
```

Comment:
Instead of hardcoding the create icon here, just import it into the story and make it as part of the children passed to the button.